src/views/mfa-verify/PushForm.js

```diff
@@ -49,6 +51,25 @@ define(['okta'], function (Okta) {
         factorName: this.model.get('factorLabel'),
         deviceName: this.model.get('deviceName')
       });
+
+      var self = this;
+      if (self.enabled) {
+        window.setTimeout(function () {
+          // trigger push once DOM is fully loaded
+          if (self.isAutoPushEnabled()) {
+            self.submit();
+          };
+        }, 0);
+      }
+    },
+    preRender: function () {
+      if (this.isAutoPushEnabled()) {
+        this.model.set('autoPush', true);
+      };
+    },
+    isAutoPushEnabled: function () {
+      return this.settings.get('features.autoPush') &&
+          CookieUtil.isAutoPushEnabled(this.options.appState.get("lastAuthResponse")._embedded.user.id);
```

Comment:
nit: would be better to put _embedded.user.id as a derived property in AppState.js (there are a bunch of examples in there, let me know if you have questions about it)